projects/plugins/jetpack/tests/e2e/lib/pages/wpcom/login.js

```diff
@@ -27,25 +27,25 @@ export default class LoginPage extends Page {
 		const continueButtonSelector = '//button[text()="Continue"]';
 		const submitButtonSelector = '//button[text()="Log In"]';
 
-		await page.type( usernameSelector, username );
-		await page.click( continueButtonSelector );
-		await page.waitForSelector( passwordSelector, { state: 'visible', timeout: 30 } );
-		// Even if we wait for the field to become visible Playwright might still type the password too fast
-		// and the first characters will miss the password field. A short wait fixes this
-		await page.waitForTimeout( 2000 );
-		await page.type( passwordSelector, password );
-		await page.click( submitButtonSelector );
-
-		await this.page.waitForNavigation( { waitUntil: 'domcontentloaded' } );
-
 		try {
```

Comment:
I'm not a fan of these retry mechanisms, but I know that sometimes there's not much else left to do. I'm curious in this case if we know why the login sometimes fails and how often does it happen. Maybe we can find something else. Is there maybe a network request that's failing?

Reply:
yeah, I generally agree with you. Although, I'd like to improve the test overall stability, and then we can dig into the actual problem here.